store/cachekv/mergeiterator.go

```diff
@@ -138,14 +138,12 @@ func (iter *cacheMergeIterator) Value() []byte {
 	// If parent is invalid, get the cache value.
 	if !iter.parent.Valid() {
 		value := iter.cache.Value()
-		iter.eventManager.EmitResourceAccessReadEvent("iterator", iter.storeKey, iter.cache.Key(), value)
```

Comment:
These events are causing memory leak during snapshot creation, it should be safe to delete since those events are not being used in critical path at all